articles/cloud-services-extended-support/deploy-template.md

```diff
@@ -104,7 +104,30 @@ This tutorial explains how to create a Cloud Service (extended support) deployme
           ] 
     ```
  
-3. Create a Network Profile Object and associate the public IP address to the frontend of the load balancer. A Load balancer is automatically created by the platform.
+3. Create a Cloud Service (Extended Support) object, adding appropriate `dependsOn` references if you are deploying Virtual Networks or Public IP within your template. 
```

Comment:
This sentence is incorrect. Please change it back to the original. The cloud service step is not required

Reply:
@surbhijain16  - Is there an improvement that makes this clearer?

The documentation most definitely includes a template at step 7 which includes a cloud service. At no stage in this documentation prior to step 7 is this cloud service added, with the networkprofile object existing as a property under this object. See example


```
 {
    "type": "Microsoft.Compute/cloudServices",
    "apiVersion": "2021-03-01",
    "name": "[variables('cloudServiceNameA')]",
    "location": "[parameters('location')]",
    "dependsOn": [
        "publicIpCopy"
    ],
    "tags": {},
    "properties": {
        "packageUrl": "[parameters('initialPackageSasUri')]",
        "configurationUrl": "[parameters('initialConfigurationSasUri')]",
        "upgradeMode": "[parameters('upgradeMode')]",
        "roleProfile": {
            "roles": "[parameters('cloudServiceARoles')]"
        },
        "networkProfile": {
            "loadBalancerConfigurations": [
                {
                    "id": "[concat(variables('resourcePrefix'), 'Microsoft.Network/loadBalancers/', variables('cloudServiceNameA'), '-lb')]",
                    "name": "[concat(variables('cloudServiceNameA'), '-lb')]",
                    "properties": {
                        "frontendIPConfigurations": [
                            {
                                "name": "[concat(variables('cloudServiceNameA'), '-lbfe')]",
                                "properties": {
                                    "publicIPAddress": {
                                        "id": "[resourceId('Microsoft.Network/publicIPAddresses/', concat(parameters('cloudServiceName'), '-' , variables('publicIpSuffixes')[0] ,'-publicip' ))]"
                                    }
                                }
                            }
                        ]
                    }
                }
            ]
        },
        "osProfile": {
            "secrets": "[parameters('cloudServiceASecrets')]"
        },
        "extensionProfile": {
            "extensions": "[parameters('cloudServiceAExtensions')]"
        }
    }
},
 ```

Reply:
Having a second read an understanding of the documentation in question - Is this note intended to communicate that users should complete an ARM export from the portal and THEN update said export by following steps 1-7?

![image](https://user-images.githubusercontent.com/5344599/118203491-8b039680-b49f-11eb-867c-63b742fdb83b.png)

If so, I can update my PR to include language to that effect.

Reply:
Hey @surbhijain16 - Is there anything I can do here?

Reply:
Hey @surbhijain16 @gachandw - Following up on this one?

Reply:
@ktoliver Anything else I can do here to grab feedback? or do I just close this item :|

Reply:
Is this note intended to communicate that users should complete an ARM export from the portal and THEN update said export by following steps 1-7? - Yes

Reply:
These changes look okay...reviewed the doc, I see you added an additional step for cloud service object and added a network profile under it
Looks good